files/auto/autopause-daemon.sh

```diff
@@ -92,6 +101,10 @@ do
     if java_clients_connected ; then
       logAutopause "Client reconnected - waiting for disconnect"
       STATE=E
+    elif [ -e /data/.skip-pause ] ; then
+      TIME_THRESH=$(($(current_uptime)+$AUTOPAUSE_TIMEOUT_EST))
```

Comment:
Resetting the threshold doesn't seem necessary, but doesn't seem bad either.

Reply:
Alright, I'll keep it as is then.